templates/emails/customer-new-account.php

```diff
@@ -32,6 +32,6 @@
 
 <?php endif; ?>
 
-	<p><?php printf( __( 'You can access your account area to view your orders and change your password here: %s.', 'woocommerce' ), make_clickable( esc_url( wc_get_page_permalink( 'myaccount' ) ) ) ); ?></p>
+	<p><?php printf( __( 'You can access your account area to view your orders and change your password here: %s.', 'woocommerce' ), esc_html( make_clickable( esc_url( wc_get_page_permalink( 'myaccount' ) ) ) ) ); ?></p>
```

Comment:
`make_clickable()` creates a URL with `<a href=""></a>` but not seems broke because `esc_html()`.